gui.sh

```diff
@@ -0,0 +1 @@
+conda deactivate & conda deactivate & python src/state_machine/src/main_gui.py
```

Comment:
why deactivate twice?

Reply:
Thats for my PC, I have to deactivate twice